src/clusterfuzz/_internal/fuzzing/corpus_manager.py

```diff
@@ -297,16 +350,49 @@ def upload_files(self, file_paths, timeout=CORPUS_FILES_SYNC_TIMEOUT):
     Returns:
       A bool indicating whether or not the command succeeded.
     """
+    # TODO(metzman): Merge this with rsync from disk when migration is complete.
     if not file_paths:
       return True
 
     # Get a new file_paths iterator where all files have been renamed to be
     # legal on Windows.
     file_paths = legalize_filenames(file_paths)
     gcs_url = self.get_gcs_url()
-    return self._gsutil_runner.upload_files_to_url(
+    result = self._gsutil_runner.upload_files_to_url(
         file_paths, gcs_url, timeout=timeout)
 
+    # Upload zipcorpus.
+    # TODO(metzman): Get rid of the rest of this function when migration is
+    # complete.
+    self._upload_to_zipcorpus(file_paths, partial=True)
+    return result
+
+  def get_zipcorpus_gcs_url(self, partial):
```

Comment:
nit: move this together with `get_zipcorpus_gcs_dir_url`

Reply:
That function is used by another caller so I can't.